### DISCUSSION TITLE
Why using Duan and Sun (2003) instead of Duan and Sun (2006)

### DISCUSSION THREAD

**Discussion by mcrossover97** [reaktoro/reaktoro#419] [Q&A]
### Description

I hope this message finds you well.

I was curious about why Reaktoro relies on the Duan and Sun (2003) model for calculating the activity of aqueous CO₂, rather than the updated version published in 2006 ([DOI: [[10.1016/j.marchem.2005.09.001])](https://doi.org/10.1016/j.marchem.2005.09.001)
Could you please clarify the rationale behind this choice?

Thank you for your time.

Best regards,
Mohammad

### What is your operating system?

None

**Comment by allanleal** [comment]
Hi @mcrossover97 - there is no rationale here. It makes sense to have the 2026 version implemented as well as it supports higher temperature/pressure conditions. Just could not find the time to add to Reaktoro. Thanks for your interest.
